pages/osx/yabai.md

```diff
@@ -5,11 +5,11 @@
 
 - Set the layout to bsp:
 
-`yabai -m config layout bsp`
+`yabai -m config layout {{bsp}}`
 
-- Set the window gap:
+- Set the window gap to 10pt:
```

Comment:
Is it pt or px? Nevertheless, it should be in this form:
```suggestion
- Set the window gap to 10 pt:
```

Reply:
The unit is usually not spaced (see any program that takes values like `10px` `10G`, `10K`). I would leave it as is.

Reply:
@mebeim Right, I deleted that part.

Reply:
It seems to be like it's pixels but on their documentation, they call it pt. That's why I went with pt. Shall I proceed with using px for the sake of unity between instructions here in tldr?

Reply:
I think just saying "pixels" is the best choice here. After all, you said they call it "pixels" in their documentation too, right?

Reply:
@BosEriko I can't find where exactly they use `pt`. Are you talking about [this](https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc) documentation?

Reply:
I found it here.
https://github.com/koekeishiya/yabai/wiki/Configuration#padding-and-gaps

Reply:
That's strange. I suppose they say `pt` to emphasize the fact that display scaling could result in different real pixel values. I don't see any mention of "pixels" in the docs though, I think we can leave it as is.

Reply:
No idea what this tool is, but a `pt` and a `px` are not actually the same thing. `px` is a pixel, and `pt` is a point - which may be a different size to a pixel IIRC.

Reply:
You are correct on using `pt` here. Thanks for contributing!